src/main/java/at/hannibal2/skyhanni/features/misc/visualwords/ModifyVisualWords.kt

```diff
@@ -62,4 +70,9 @@ object ModifyVisualWords {
             SkyHanniMod.configManager.saveConfig(ConfigFileType.VISUAL_WORDS, "Migrate visual words")
         }
     }
+
+    @SubscribeEvent
+    fun onConfigLoad(event: ConfigLoadEvent) {
+        SkyHanniMod.feature.dev.debug.neverFunnyTime.onToggle { textCache.clear() }
```

Comment:
this is not needed, if the isAprilFoolsDay is updated.

Reply:
well it didnt work without it for some reason

Reply:
because the isAprilFoolsDay was/is wrong

Reply:
i see, committed changes